### PR TITLE
Update to various Documents

### PR DESCRIPTION
**Pull request by bmfmancini**
Hey Guy 

Just added some pictures to the automation doc and added more in-depth instructions

### CODE REVIEW

File: Automation-Networks.md [hunk 1]
```diff
@@ -2,11 +2,53 @@
 
 This section will describe **Automation Networks** in Cacti.
 
-![Automation Networks](images/automation-networks.png)
+Adding a network to scan in the automation plugin is easy  on the main console click Automation
+once on the below page click the + on the top right of the page 
+
+![Automation Networks](images/automation-network-main.png)
+
+<br>
```
Comment (netniV) [inline Automation-Networks.md]: Do not use inline HTML codes

File: Automation-Networks.md [hunk 2]
```diff
@@ -2,11 +2,53 @@
...
+<br>
+<br>
+
+you will now see the below page for example if you want to scan 192.168.1.0/24 you would enter that in the subnet range textbox<br>
+you will need to note the subnet in CIDR format<br>
```
Comment (netniV) [inline Automation-Networks.md]: Do not use inline HTML codes

File: Automation-Networks.md [hunk 3]
```diff
@@ -2,11 +2,53 @@
...
+
+Other important options are 
+<br>
+Schedule type - How often you want to scan this subnet for devices<br>
```
Comment (netniV) [inline Automation-Networks.md]: 1. Add a - (minus) to the beginning of the line to denote a bullet list
   or convert to an MDL table
2. Do not use inline HTML codes

Comment (bmfmancini) [reply Automation-Networks.md]: Sounds Good! 

I am ok with those changes

**Comment by bmfmancini** [comment]
I have made the changes let me know if this is good

**Comment by netniV** [comment]
You are still using inline html tags which should not be present within the MD files.  Only GFM (Github Flavoured Markdown) should be used. See our [Documentation Standards](https://github.com/Cacti/documentation/blob/develop/Standards-Documentation.md)

**Comment by bmfmancini** [comment]
Removed all HTML tags on both automation and Ubuntu installation docs

**Comment by bmfmancini** [comment]
More Doc updates

**Comment by netniV** [comment]
There is just one more change needed, and then we can merge this.

```
Not Linked: Installing-Under-Ubuntu-Debian.md
1 unlinked files
```

This means you've added a new file and there is no reference to it in README.md so if you can add one there travis should be happy.

**Comment by bmfmancini** [comment]
Oh right ok let me add that

**Comment by bmfmancini** [comment]
Ok committed the changes

**Comment by netniV** [comment]
As always, get Travis to move forward it finds something else.  https://travis-ci.org/Cacti/documentation/builds/534163834?utm_source=github_status&utm_medium=notification

If you want to try and catch these before committing, install MDL (MarkdownLint) and use the two .sh scripts in the bin folders as that is pretty much what travis is doing.

Making good progress though!

**Comment by netniV** [comment]
This was closed before it was merged, @bmfmancini ?

**Comment by bmfmancini** [comment]
Oops lol sorry about that

**Comment by bmfmancini** [comment]
Hey Guys 

Looks like all checks have passed
should be ready to merge now

**Pull request merged**